samples/ingestion/ingestion-client/Setup/guide.md

```diff
@@ -159,6 +159,24 @@ There are several containers to distinguish between the various outputs. We sugg
 
 By default, the ARM template uses the newest version of the Ingestion Client which can be found in this repository. To use a custom version, edit the paths to the binaries inside the deployment template to point to a custom published version. You can find our published binaries [here](https://github.com/Azure-Samples/cognitive-services-speech-sdk/releases?q=ingestion+client&expanded=true).
 
+When using Windows for development, for any changes to the resources, you need to update the `main.bicep` and generate the corresponding `main.json`. If `main.json` is not in sync with the `main.bicep`, your build will fail.
+
+To generate `main.json`, you need to make sure Azure CLI & Bicep CLI are installed. Execute below commands to install:
+To install Azure CLI:
+``` bash
+curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash
+```
+To install Bicep CLI:
+```bash
+az bicep install
+```
+Once Bicep CLI is installed execute the below command to generate `main.json`.
+``` bash
+az bicep build --file ./samples/ingestion/ingestion-client/infra/main.bicep
+```
+
+When using Linux, any changes to `main.bicep`, every commit will generate the corresponding `main.json`.
```

Comment:
is it also generating main.json if you commit from linux without using devcontainers?

Reply:
Yes it does.